### PR TITLE
feat: add helm repo install check

### PR DESCRIPTION
**Pull request by umagnus**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

feat: add helm repo install check

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: hack/verify-helm-chart-index.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+function check_url() {  
+    url=$1
+    result=$(curl -I -m 5 -s -w "%{http_code}\n" -o /dev/null $1)
+    if [ $result -eq 200 ]
+    then
+        echo "$1 is ok."
+    else
+        echo "$1 is fail."
+        echo "rm "${PKG_ROOT}${url#*master}
+        rm -rf ${PKG_ROOT}${url#*master}
```
Comment (andyzhangx) [inline hack/verify-helm-chart-index.sh:32]: this is just a check tool, if there is invalid url, just print it and report error, don't update original index file.

Comment (umagnus) [reply hack/verify-helm-chart-index.sh:32]: ok，I will remove it.

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**